package.json

```diff
@@ -18,7 +18,7 @@
   "licenses": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "basisjs": "basisjs/basisjs#1.5",
+    "basisjs": "basisjs/basisjs#v1.5.1",
```

Comment:
no. It doesn't work. At least with my version of npm:

npm ERR! git rev-list -n1 1.5.1: fatal: ambiguous argument '1.5.1': unknown revision or path not in the working tree.